packages/forms-web-app/src/lib/client-side/google-tag-manager.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-env browser */
```

Comment:
Ideally we should have unit tests for this file. The test coverage is failing and adding these tests should make that pass.

Reply:
Have added tests.